docs/products/kafka/kafka-connect/howto/configure-aws-secrets-manager.md

```diff
@@ -0,0 +1,256 @@
+- `Authorization`: Header for authentication. Replace `YOUR_BEARER_TOKEN` with your
+  actual [Aiven API token](/docs/platform/howto/create_authentication_token).
+- `Content-Type`: Specifies that the request body is in JSON format.
+- `auth_method`: Authentication method used by AWS Secrets Manager. In this case,
+  it is `credentials`.
+- `region`: AWS region where your secrets are stored.
+- `access_key`: Your AWS access key.
+- `secret_key`: Your AWS secret key.
+
+</TabItem>
+<TabItem value="cli" label="CLI">
+
+Add AWS Secrets Manager using the [Aiven CLI](/docs/tools/cli):
+
+```bash
+avn service update demo-kafka-service \
+  -c secret_providers='[
+    {
+      "name": "aws",
+      "aws": {
+        "auth_method": "credentials",
+        "region": "your-aws-region",
+        "access_key": "your-aws-access-key",
+        "secret_key": "your-aws-secret-key"
+      }
+    }
+  ]'
+
+```
+
+Parameters:
+
+- `project`: Name of your Aiven project.
+- `service`: Name of your Aiven Kafka service.
+- `name`: Name of the secret provider. In this case, `aws`.
+- `auth_method`: Authentication method used by AWS Secrets Manager. In this case, it is credentials.
+- `region`: AWS region where your secrets are stored.
+- `access_key`: Your AWS access key.
+- `secret_key`: Your AWS secret key.
+
+</TabItem>
+</Tabs>
+
+## Reference secrets in connector configurations
+
+After configuring the secret providers, you can reference secrets in your
+connector configurations.
+
+### JDBC sink connector
+
+<Tabs groupId="reference-secrets-sink">
+<TabItem value="api" label="API" default>
+
+Reference secrets in the JDBC sink connector configuration using the following
+API request:
+
+```sh
+curl -X POST https://api.aiven.io/v1/project/<project_name>/service/<service_name>/connectors \
+  -H "Authorization: Bearer <your_auth_token>" \
+  -H "Content-Type: application/json" \
+  -d '{
+        "name": "your-sink-connector-name",
+        "connector.class": "io.aiven.connect.jdbc.JdbcSinkConnector",
+        "connection.url": "jdbc:postgresql://host:port/dbname?user=${aws:path/to/secret:username}&password=${aws:path/to/secret:password}&ssl=require",
+        "topics": "your-topic",
+        "auto.create": "true"
+      }'
+```
+
+Parameters:
+
+- `name`: Name of the connector.
+- `connector.class`: Specifies the connector class to use, in this case,
+  `io.aiven.connect.jdbc.JdbcSinkConnector`.
+- `connection.url`: JDBC connection URL, with placeholders for the username and password
+  retrieved from AWS Secrets Manager.
+- `topics`: Apache Kafka topic where the data can be sent.
+- `auto.create`: If `true`, the connector automatically creates the table in the
+  target database if it does not exist.
+
+</TabItem>
+<TabItem value="cli" label="CLI">
+
+Reference secrets in the JDBC sink connector configuration using the following
+CLI command:
+
+```bash
+avn service connector create                                    \
```

Comment:
I think this is not valid cli usage. Have a look at: https://aiven.io/docs/tools/cli/service/connector.

```bash
avn service connector create --help
usage: avn service connector create [-h] [--project PROJECT] service_name connector_config

Create a Kafka connector

positional arguments:
  connector_config   JSON string or path (preceded by '@') to a JSON configuration file
  service_name       Service name

options:
  -h, --help         show this help message and exit
  --project PROJECT  Project name to use, default None
```